src/Microsoft.IdentityModel.Tokens/opensource/json/ConstructorHandling.cs

```diff
@@ -28,7 +28,7 @@ namespace Microsoft.IdentityModel.Json
     /// <summary>
     /// Specifies how constructors are used when initializing objects during deserialization by the <see cref="JsonSerializer"/>.
     /// </summary>
-    internal enum ConstructorHandling
+    public enum ConstructorHandling
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

These need to be public now?

Reply:
My bad, need to make these internal.
Will reissue the PR.